bazel/bazelrcs/remote_caching_ro.bazelrc

```diff
@@ -0,0 +1,2 @@
+build --config=remote_caching_ro
```

Comment:
Even though we already explain this in `.bazelrc`, I think it wouldn't hurt to also explain here that this is supposed to be templated and written to `remote-cache.bazelrc` where it is imported by `.bazelrc`

Reply:
Comments added